bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.html

```diff
@@ -81,8 +81,16 @@
               <div fxFlex fxLayout="row" fxLayoutAlign="flex-start center" fxLayoutGap="15px">
                 <div class="mt-1 mb-1 sequence-title" fxLayout="row" fxLayoutAlign="start center">
                   <p class="m-0">
-                    <span class="bold" [textContent]="currentSequence.name"></span>&nbsp;
-                    <span [textContent]="currentSequence.getStatus()"></span>
+                    <span class="bold" [textContent]="currentSequence.name"></span>
+                    of
+                    <span
```

Comment:
This could be short handed to:
```
<span class="bold"
  [textContent]="currentSequence.getShortImageName() ? currentSequence.getShortImageName() : currentSequence.service"
></span>
&nbsp;<span [textContent]="currentSequence.getStatus()"></span>
```

Then the whole `ng-template` block would not be needed.

Reply:
Hi Elisabeth, I've refactored the element as requested here: https://github.com/keptn/keptn/pull/7283/commits/1ce18cc7b0f5f16f2aea830bc8dfe6d37a31de13. Thanks!